### PR TITLE
update(cassandra): getting started

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/DOC-796

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/products/cassandra/get-started.md [hunk 1]
```diff
@@ -4,34 +4,412 @@ sidebar_label: Get started
...
+using the
+[the `cqlsh` client](https://cassandra.apache.org/doc/stable/cassandra/tools/cqlsh.html).
+</TabItem>
+</Tabs>
 
-- To preview the full list of default flags, run `avn service create -h`.
-- To list the type-specific options, run `avn service types -v`.
+:::tip
+Discover more tools for connecting to Aiven for Apache Cassandra in
+[Connect to Aiven for Apache Cassandra®](/docs/products/clickhouse/howto/list-connect-to-service).
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/get-started.md:340]: > /docs/products/clickhouse

probably typo, should be Cassandra? the link leads to Clickhouse

File: docs/products/cassandra/get-started.md [hunk 2]
```diff
@@ -4,34 +4,412 @@ sidebar_label: Get started
...
+   organization > project > Aiven for Apache Cassandra service.
+1. On the <ConsoleLabel name="overview"/> page of your service, click
+   **Quick connect**.
+1. In the **Connect** window, select a tool or language to connect to your service, follow
+   the connection instructions, and click **Done**.
+
+   ```bash
+   # Sample command for connecting with the cqlsh client
+
+   ./cqlsh --ssl -u USER_NAME -p PASSWORD HOST_NAME PORT_NUMBER
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/get-started.md:318]: this command should be the same both for Console and Terraform? this version is missing the host and port, so most likely won't work for the customer?

File: docs/products/cassandra/get-started.md [hunk 3]
```diff
@@ -4,34 +4,412 @@ sidebar_label: Get started
...
+:::
+
+## Create keyspaces and tables
+
+Use `cqlsh` to create a keyspace and a table where you can insert your data next.
+
+1. Create a keyspace, for example `library`:
+
+   ```sql
+   CREATE KEYSPACE IF NOT EXISTS library WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : '3' };
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/get-started.md:350]: could you change the example to use NetworkTopologyStrategy? I want to sunset SimpleStrategy eventually, the replication part should look like this:
```
... REPLICATION = { 'class': 'NetworkTopologyStrategy', 'aiven': 3 }
```
where 'aiven' is the default datacenter name for Aiven for Cassandra services.

File: docs/products/cassandra/get-started.md [hunk 4]
```diff
@@ -4,34 +4,412 @@ sidebar_label: Get started
...
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('12345', 55, 'spirituality_religions', 'available', toTimeStamp(now()));
+INSERT INTO library.book_tracker (book_id, item_count, genre, status, last_update_timestamp) VALUES ('67890', 3, 'adventure', 'unavailable', toTimeStamp(now()));
+```
+
+:::tip
+For importing large amount of data or for batch data load, see:
+
+- [Use DSBULK to load, unload, and count data](/docs/products/cassandra/howto/use-dsbulk-with-cassandra)
+- [Migrate to Aiven with the ZDM Proxy](/docs/products/cassandra/howto/zdm-proxy)
+- [Migrate to Aiven with `sstableloader`](https://aiven.io/blog/aiven-now-supports-sstableloader)
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/get-started.md:375]: maybe just remove the sstableloader option? the link leads to a blog post, which links to /dev/null ("the complete procedure can be found here" link leads to somewhere that does not exist, probably the old article). sstableloader migration IMHO is legacy, people are not likely to want this, I'd just leave the `dsbulk` and ZDM proxy options

File: docs/products/cassandra/get-started.md [hunk 5]
```diff
@@ -4,34 +4,412 @@ sidebar_label: Get started
...
 
 :::
+
+## Read data
+
+Use `cqlsh` to read data from the `book_tracker` table, for example:
+
+- ```sql
+  SELECT * FROM library.book_tracker;
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/get-started.md:384]: could you add a `WHERE` clause to the queries? the "SELECT everything" is a bad antipattern in Cassandra world, it requires the "ALLOW FILTERING" to suppress the built-in Cassandra safety mechanisms that fail these kinds of queries; a good query would look like `SELECT ... WHERE book_id = 01234`

File: docs/products/cassandra/get-started.md [hunk 6]
```diff
@@ -4,34 +4,412 @@ sidebar_label: Get started
...
+
+  book_id | genre
+  --------+--------
+  56789   | romance
+  ```
+
+## Related pages
+
+- [Supported Aiven for Apache Cassandra versions](/docs/platform/reference/eol-for-major-versions#h_0f2929c770)
+- [Aiven for Apache Cassandra backups](/docs/platform/concepts/service_backups#aiven-for-apache-cassandra)
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/get-started.md:414]: (could be a separate PR) in that backup article, maybe mention that `backup_hour` and `backup_minute` are available for Cassandras and allow the customer to set _when_ the backup starts within the 24 hours? something a-la "if you want to avoid spending extra resources on backups during peak hours, you can set backup_hour in advanced user_config to move the backup outside the peak hours"

**Review by dmitry-potepalov — CHANGES_REQUESTED** [review]
overall pretty good :+1: some details needs to be improved, outlined in the comments

File: docs/products/cassandra/get-started.md [hunk 7]
```diff
@@ -4,34 +4,412 @@ sidebar_label: Get started
...
+Create an Aiven for Apache Cassandra service using the Aiven Operator for Kubernetes.
+
+1. [Get authenticated and authorized](https://aiven.github.io/aiven-operator/authentication.html).
+1. Create file `cassandra-sample.yaml` with the following content:
+
+    ```yaml
+    apiVersion: aiven.io/v1alpha1
+    kind: Cassandra
+    metadata:
+    name: cassandra-sample
```
Comment (pavdmyt) [inline docs/products/cassandra/get-started.md:123]: Indentation is not preserved, `name` field should be nested under the `metadata`. Check https://aiven.github.io/aiven-operator/resources/cassandra.html for example.

**Review by pavdmyt — DISMISSED** [review]
Some comments, mostly about formatting.

File: docs/products/cassandra/get-started.md (hunk 2 above)
Comment (wojcik-dorota) [reply docs/products/cassandra/get-started.md:318]: In this command here, I've included placeholders (`HOST_NAME` and `PORT_NUMBER`) so that the user can replace them with the actual values. Same for `USER_NAME` and `PASSWORD` in this command. In any case, this command is only included as an example coz the actual guidance we're providing here is "Follow the steps 1-3".
In the Terraform flow, the corresponding command differs coz it's using the envir variables we saved specifically for the TF flow purposes (see lines 105-108) so they are not available in the console flow.

File: docs/products/cassandra/get-started.md (hunk 5 above)
Comment (wojcik-dorota) [reply docs/products/cassandra/get-started.md:384]: Thanks @dmitry-potepalov ! There are already two `WHERE` examples that follows this one. Also, I tested this example and it worked fine for me as is. Please let me know if you still think this example is not ok and I'll remove it (adding another `WHERE` example would only add redundancy I guess).

File: docs/products/cassandra/get-started.md (hunk 6 above)
Comment (wojcik-dorota) [reply docs/products/cassandra/get-started.md:414]: thanks! excellent point! updating on this PR...

File: docs/products/cassandra/get-started.md (hunk 5 above)
Comment (dmitry-potepalov) [reply docs/products/cassandra/get-started.md:385]: Let's leave the first "select all" example and change the `WHERE` clause to reference the primary key of the table (book_id). The queries that require `ALLOW FILTERING` are an antipattern we don't want to promote, they'll get progressively slower and slower as the cluster grows since they don't allow the database to select the right node for the query

Comment (wojcik-dorota) [reply docs/products/cassandra/get-started.md:385]: I updated the first example but I'm not sure how to modify the other ones. Could you please comment inline on each of them and provide your suggestions for improvement?

**Review by pavdmyt — APPROVED** [review]
LGTM from me.

File: docs/products/cassandra/get-started.md [hunk 8]
```diff
@@ -4,34 +4,410 @@ sidebar_label: Get started
...
+- ```sql
+  SELECT * FROM library.book_tracker WHERE book_id = '01234';
+
+  book_id | genre                  | item_count | last_update_timestamp           | status
+  --------+------------------------+------------+---------------------------------+------------
+  01234   |             true_crime |          6 | 2024-07-31 11:21:14.263000+0000 |   available
+  ```
+
+- ```sql
+  SELECT * FROM library.book_tracker WHERE status = 'available' ALLOW FILTERING;
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/get-started.md:393]:
```
SELECT * FROM library.book_tracker WHERE book_id = '01234'
```

File: docs/products/cassandra/get-started.md [hunk 9]
```diff
@@ -4,34 +4,410 @@ sidebar_label: Get started
...
+
+  book_id | genre                  | item_count | last_update_timestamp           | status
+  --------+------------------------+------------+---------------------------------+----------
+  01234   |             true_crime |          6 | 2024-07-31 11:21:14.263000+0000 | available
+  12345   | spirituality_religions |         55 | 2024-07-31 11:24:20.373000+0000 | available
+  ```
+
+- ```sql
+  SELECT book_id, genre FROM library.book_tracker WHERE item_count > 10 AND status = 'unavailable' ALLOW FILTERING;
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/get-started.md:402]:
```
SELECT genre, status FROM library.book_tracker WHERE book_id IN ('01234', '12345')
```

**Review by dmitry-potepalov — APPROVED** [review]
LGTM

File: docs/products/cassandra/get-started.md [hunk 10]
```diff
@@ -4,34 +4,418 @@ sidebar_label: Get started
...
+   CASSANDRA_HOST="$(terraform output -raw cassandra_service_host)"
+   CASSANDRA_PORT="$(terraform output -raw cassandra_service_port)"
+   CASSANDRA_USER="$(terraform output -raw cassandra_service_username)"
+   CASSANDRA_PASSWORD="$(terraform output -raw cassandra_service_password)"
+   ```
+
+</TabItem>
+<TabItem value="3" label="K8s">
+
+Create an Aiven for Apache Cassandra service using the Aiven Operator for Kubernetes.
```
Comment (ArthurFlag) [inline docs/products/cassandra/get-started.md:114]: We do not have a similar intro sentence for TF, let's do similiar things? Either delete this line or add one to the TF tab?

File: docs/products/cassandra/get-started.md [hunk 11]
```diff
@@ -4,34 +4,418 @@ sidebar_label: Get started
...
+   ```
+
+1. Review the resource you created with the following command:
+
+   ```shell
+   kubectl get cassandras cassandra-sample
+   ```
+
+The resource can stay in the `REBUILDING` state for a couple of minutes. Once the state
+changes to `RUNNING`, you are ready to access it.
```
Comment (ArthurFlag) [inline docs/products/cassandra/get-started.md:157]: We do not say this for TF, but I suppose it's equally correct and should appear in the TF tab too?

File: docs/products/cassandra/get-started.md [hunk 12]
```diff
@@ -4,34 +4,418 @@ sidebar_label: Get started
...
+   the <ConsoleLabel name="Services"/> page.
+1. On the <ConsoleLabel name="overview"/> page, select <ConsoleLabel name="service settings"/>
+   from the sidebar.
+1. In the **Advanced configuration** section, make changes to the service configuration.
+</TabItem>
+<TabItem value="2" label="Terraform">
+
+:::note
+Your changes can force the recreation of the `aiven_cassandra` resource.
+:::
```
Comment (ArthurFlag) [inline docs/products/cassandra/get-started.md:182]: I imagine this also applies to the K8s tab?

File: docs/products/cassandra/get-started.md [hunk 13]
```diff
@@ -4,34 +4,418 @@ sidebar_label: Get started
...
+   ```
+
+1. Review the resource you updated with the following command:
+
+   ```shell
+   kubectl describe cassandra.aiven.io cassandra-sample
+   ```
+
+The resource can stay in the `REBUILDING` state for a couple of minutes. Once the state
+changes to `RUNNING`, you are ready to access it.
```
Comment (ArthurFlag) [inline docs/products/cassandra/get-started.md:297]: Same comment here about TF/K8s symmetry.

File: docs/products/cassandra/get-started.md [hunk 14]
```diff
@@ -4,34 +4,418 @@ sidebar_label: Get started
...
+
+See the available configuration options in
+[Aiven Operator for Kubernetes®: Cassandra](https://aiven.github.io/aiven-operator/api-reference/cassandra.html).
```
Comment (ArthurFlag) [inline docs/products/cassandra/get-started.md:300]: The TF equivalent seems to be an admonition at the top of the tab. Should we align the tabs on one approach/pattern?

File: docs/products/cassandra/get-started.md [hunk 15]
```diff
@@ -4,34 +4,418 @@ sidebar_label: Get started
...
-   --service-type cassandra             \
-   --cloud CLOUD_AND_REGION             \
-   --plan PLAN                          \
-   --project PROJECT_NAME
+./cqlsh --ssl --host=$CASSANDRA_HOST --port=$CASSANDRA_PORT --user=$CASSANDRA_USER --password=$CASSANDRA_PASSWORD
 ```
 
-:::note
+</TabItem>
+<TabItem value="3" label="cqlsh client">
```
Comment (ArthurFlag) [inline docs/products/cassandra/get-started.md:331]: Since all the tabs on the page are in the same group (group1) this tab with value 3, syncs with tabs with value 3 above, but those tabs are about K8s, so this tab needs a new value to be independant.

```suggestion
<TabItem value="cqlsh" label="cqlsh client">
```

**Review by ArthurFlag — CHANGES_REQUESTED** [review]
Some harmony improvement suggestions, but it looks good otherwise 👍 🙌

File: docs/platform/concepts/service_backups.md [hunk 16]
```diff
@@ -261,11 +261,16 @@ For more information, see:
 
 ### Aiven for Apache Cassandra®
 
-Aiven for Apache Cassandra backups are taken every 24 hours. The
-point-in-time recovery (PITR) feature is currently not available.
+Aiven for Apache Cassandra backups are taken every 24 hours. You can choose when a
+backup starts within the 24 hours and, for example, move the backup outside peak hours to
+save resources. To configure the `backup_hour` and `backup_minute` settings, go to
+the [Aiven Console](https://console.aiven.io) > your Aiven for Apache Cassandra service >
+<ConsoleLabel name="backups"/> > <ConsoleLabel name="actions"/> > **Configure backup settings**.
```
[the remark this answers is not in the document]
Comment (wojcik-dorota) [reply docs/platform/concepts/service_backups.md:268]: Will create a ticket and move this info to the new page as soon as created
